synapse/lib/ast.py

```diff
@@ -521,19 +521,17 @@ class VarEvalOper(Oper):
     $foo.bar("baz")
     '''
     async def run(self, runt, genr):
+        id = s_common.guid(str(self))[:4]
```

Comment:
this seems like vestigial debug code?  also, id is a python builtin and should be avoided :D

Reply:
yes debug code